app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
   belongs_to :user
```

Comment:
[REQUIRED]This should be `belongs_to :author, class_name: 'User'`. This helps differentiate between the different types of users